examples/dem/3d-rotating-drum/post_processing_rotating_drum.py

```diff
@@ -105,5 +105,5 @@
 plt.plot(y_graph * 1.214749159, y_graph, "-.", label=r"$\omega r$")
 plt.legend()
 plt.title(f"Average velocity parallel to the free surface \ndepending on the depth from the center of the cylinder")
-plt.show()
+plt.savefig("lethe-rotating-drum-comparison.pdf")
```

Comment:
I would instead keep the plt.show instead of making it by default a pdf file. You can use the argparser --validate instead like in the other script to alternate between one or the other. You can look at the other validate scripts.